lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -324,14 +327,39 @@ export class StandardController implements IController {
             this.initialized
         );
 
+        // Throw an error if user has set OIDCOptions without being in OIDC protocol mode
+        if(this.config.auth.protocolMode !== ProtocolMode.OIDC && 
+            this.config.auth.OIDCOptions) {
+                throw ClientConfigurationError.createCannotSetOIDCOptionsError();
+        }
+
+        let found_hash = hash;
+        
+        if(this.config.auth.protocolMode === ProtocolMode.OIDC && 
+            this.config.auth.OIDCOptions?.serverResponseType === ServerResponseType.QUERY) {
+                /**
+                 * Check from ?code to make sure we don't get a random query string
+                 * until # since some IDPs add stuff that doesn't concern MSAL after the # 
```

Comment:
Wording.  "add stuff that doesn't concern MSAL..."  
Suggest something more like:  extract hash between '?code=' and '#' if trailing '# is present.